doc/flame_fire_atlas/fire_atlas.md

```diff
@@ -0,0 +1,86 @@
+# FireAtlas
+
+FireAtlas is a mapping file that can hold several Sprites and SpriteAnimations.
```

Comment:
This is a description of the file that FireAtlas produces, not a description of what FireAtlas is.
You can try it out here if you haven't:
https://fire-atlas.flame-engine.org/

Reply:
it is object that represents the mapping file, right ?

Reply:
No, FireAtlas is a tool for handling texture atlases, the file is just the output from the tool. So it's the tool that should be described here.